heartwood-version

```diff
@@ -0,0 +1 @@
+a6a3290833f6da4a7785a426bed342df069ce47f
```

Comment:
Could we move this to `tests/support/heartwood-version`? Feel like it belongs there instead of the top-level, since it's only used by tests.

Reply:
Yeah was my first move too, but the top-level was also nice, since we have all the nvmrc and config files there

Reply:
c33c61048723439f0db577a8f05ced0c9d0474c2